params/config.go

```diff
@@ -248,6 +251,26 @@ var (
 		Aura:                &AuRaConfig{},
 	}
 
+	FermionChainConfig = &ChainConfig{
+		ChainName:           FermionChainName,
+		ChainID:             big.NewInt(102),
+		HomesteadBlock:      big.NewInt(0),
+		DAOForkBlock:        nil,
+		DAOForkSupport:      false,
+		EIP150Block:         big.NewInt(0),
+		EIP155Block:         big.NewInt(0),
+		EIP158Block:         big.NewInt(0),
+		ByzantiumBlock:      big.NewInt(5067000),
+		ConstantinopleBlock: big.NewInt(9200000),
+		PetersburgBlock:     big.NewInt(10255201),
+		IstanbulBlock:       big.NewInt(14111141),
+		MuirGlacierBlock:    nil,
+		BerlinBlock:         big.NewInt(24770900),
+		LondonBlock:         big.NewInt(26741100),
```

Comment:
Are you sure that you wanna enable London features after 4 years from network start?